pom.xml

```diff
@@ -289,6 +289,11 @@
                 <version>1.1.1</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.crypto</groupId>
+                <artifactId>org.wso2.carbon.crypto.impl</artifactId>
+                <version>1.1.13</version>
```

Comment:
Move this to a property

Reply:
Resolved by https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/116/commits/baf54ab3dc0b233d547617eec89f5eefc860ff90